source/features/index.tsx

```diff
@@ -244,6 +248,19 @@ const add = async (id: FeatureID, ...loaders: FeatureLoader[]): Promise<void> =>
 	}
 };
 
+const addCssFeature = async (id: FeatureID, loaders: CssInternalRunConfig): Promise<void> => {
+	async function cssOnlyFeature(featureName: string): Promise<void> {
+		await Promise.resolve(); // The event fires a bit before the document body loaded
+		document.body.classList.add('rgh-' + featureName);
+	}
+
+	void add(id, {
+		...loaders,
+		awaitDomReady: false,
+		init: () => void cssOnlyFeature(id)
```

Comment:
Just inline it, don’t call a secondary function.

Reply:
Forgot to make it a draft. I did not have time to finish it yet.